src/python/pants/backend/jvm/targets/jar_dependency.py

```diff
@@ -51,16 +111,34 @@ def __new__(cls, org, name, rev=None, force=False, ext=None, url=None, apidocs=N
       the dependency itself should be downloaded and placed on the classpath
     :param list excludes: Transitive dependencies of this jar to exclude.
     :type excludes: list of :class:`pants.backend.jvm.targets.exclude.Exclude`
+    :param string base_path: absolute base path that a relative file url is based from.
```

Comment:
so are you ok with the new param? it's a change for devs (compatible) but not for users, no build file api change. doc should have already reflected this.